app/src/main/java/org/simple/clinic/forgotpin/confirmpin/ForgotPinConfirmPinScreen.kt

```diff
@@ -2,7 +2,61 @@ package org.simple.clinic.forgotpin.confirmpin
 
 import android.content.Context
 import android.util.AttributeSet
+import android.widget.ImageButton
 import android.widget.RelativeLayout
+import android.widget.TextView
+import com.jakewharton.rxbinding2.view.RxView
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers.mainThread
+import io.reactivex.schedulers.Schedulers.io
+import kotterknife.bindView
+import org.simple.clinic.R
+import org.simple.clinic.activity.TheActivity
+import org.simple.clinic.router.screen.ScreenRouter
+import org.simple.clinic.widgets.ScreenCreated
+import org.simple.clinic.widgets.UiEvent
+import javax.inject.Inject
 
 class ForgotPinConfirmPinScreen(context: Context, attributeSet: AttributeSet?) : RelativeLayout(context, attributeSet) {
+
+  @Inject
+  lateinit var controller: ForgotPinConfirmPinScreenController
+
+  @Inject
+  lateinit var screenRouter: ScreenRouter
+
+  private val backButton by bindView<ImageButton>(R.id.forgotpin_back)
```

Comment:
Hey, the IDs use the same prefix for both the screens: `forgotpin_`

Reply:
Copy-Pasta. Will fix in the final PR for this feature.